drools-wb-screens/drools-wb-scorecard-xls-editor/drools-wb-scorecard-xls-editor-backend/src/main/java/org/drools/workbench/screens/scorecardxls/backend/server/ScoreCardXLSServiceImpl.java

```diff
@@ -249,15 +247,24 @@ public Path copy( final Path path,
     @Override
     public List<ValidationMessage> validate( final Path path,
                                              final Path resource ) {
+        InputStream inputStream = null;
```

Comment:
Since I see Java8-specific code elsewhere in this PR, maybe we want to convert this into try-with-resources?